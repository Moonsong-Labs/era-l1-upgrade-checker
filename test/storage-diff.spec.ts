import {describe, it} from "vitest";
import {execAsync} from "./util";

describe("storage-diff", () => {
  it("prints all the information for the mini upgrade", async () => {
    const {stdout} = await execAsync("pnpm validate storage-diff reference/1699353977-boojum");

    console.log(stdout)
  })
})