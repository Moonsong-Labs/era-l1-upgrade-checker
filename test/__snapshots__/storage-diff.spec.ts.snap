// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`storage-diff > matches snapshot 1`] = `
"- Gathering contract data: OK
No storage changes when executing upgrade.
"
`;
